src/Futhark/LSP/Handlers.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Futhark.LSP.Handlers (handlers) where
+
+import Control.Concurrent.MVar (MVar)
+import Control.Lens ((^.))
+import Control.Monad.IO.Class (MonadIO (liftIO))
+import Futhark.LSP.Compile (tryReCompile, tryTakeStateFromMVar)
+import Futhark.LSP.State (State (..))
+import Futhark.LSP.Tool (getHoverInfoFromState)
+import Futhark.Util (debug)
+import Futhark.Util.Pretty (pretty)
+import Language.LSP.Server (Handlers, LspM, getVersionedTextDoc, notificationHandler, requestHandler)
+import Language.LSP.Types
```

Comment:
This is still importing lots of staff. There would be `27` more lines of import if I explicit import everything

Reply:
Yes, that's too much.  But you can consider a qualified import later if things get out of hand (`import qualified Language.LSP.Types as LSP`).